### PR TITLE
[Improve] Login Screen UI Update

### LINKED ISSUE #2793: Update Login Screen
https://discord.com/channels/@me/920267778190086205/1174274661077499967

### PR DESCRIPTION
**Pull request by aliseyalvi**
### What does this PR?
Updated login sceen UI according to ecency web login UI

### Where should the reviewer start?
Go to Login Screen on mobile app and check updated UI

### Issue number
fixes #2793 

### Screenshots/Video
image

### CODE REVIEW

**Comment by aliseyalvi** [comment]
image

image

**Comment by aliseyalvi** [comment]
@noumantahir fixed [this issue](https://discord.com/channels/@me/920267778190086205/1178935396692934657) as well

File: src/components/orDivider/orDividerView.tsx [hunk 1]
```diff
@@ -0,0 +1,18 @@
...
+import styles from './orDividerStyles';
+
+interface OrDividerProps {
+  containerStyle?: ViewStyle;
+}
+const OrDivider = ({ containerStyle }: OrDividerProps) => {
+  return (
+    <View style={[styles.dividerContainer, containerStyle]}>
+      <View style={[styles.divider, styles.leftDivider]} />
+      <Text style={styles.orText}>OR</Text>
```
Comment (noumantahir) [inline src/components/orDivider/orDividerView.tsx:12]: use intl

**Review by noumantahir — CHANGES_REQUESTED** [review]
Check how the input skips characters some times... this has been there for quite some time now... if we are improving Login, I think this can be addressed as well...

https://github.com/ecency/ecency-mobile/assets/6298342/fe90d266-34e8-4e55-b923-2ff69b6e943b

Do also check one comment in PR thread. Other than that I think we can wrap this up and move to signup screen next.

**Comment by aliseyalvi** [comment]
@noumantahir 

- converted login screen class component to functional component and hooks
- updated debounce logic which was causing characters glitch
- used intl

https://github.com/ecency/ecency-mobile/assets/48380998/c443a1d5-afdb-4f60-9faf-56f15655e7a1

**Review by noumantahir — APPROVED** [review]
lgtm

**Review by feruzm — APPROVED** [review]
lgtm, would be better if we converted those modified components/pages to typescript.

**Pull request merged**